pkg/util/k8s/k8s.go

```diff
@@ -611,8 +611,10 @@ func CreateOrUpdateCSIDriver(
 		return err
 	}
 
-	// Copy the fsGroupPolicy as it is automatically populated by k8s
+	// Copy the fields that are automatically populated by k8s
 	driver.Spec.FSGroupPolicy = existingDriver.Spec.FSGroupPolicy
+	driver.Spec.RequiresRepublish = existingDriver.Spec.RequiresRepublish
+	driver.Spec.StorageCapacity = existingDriver.Spec.StorageCapacity
```

Comment:
Without this change, the CSI driver object keeps getting updated regularly because there is difference between the object we create and the one present in k8s due to default values.